src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx

```diff
@@ -23,37 +19,34 @@ Complete the steps below to collect Kafka metrics from Confluent.
+* You must have [Docker Compose](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/confluentcloud) installed
+* You will need to generate a [TLS authentication key](https://docs.confluent.io/platform/current/kafka/encryption.html) from your [Confluent Cloud account](https://www.confluent.io/get-started/).  
 
-<Callout variant="important">
-  New Relic supports the OpenTelemetry community by contributing our work upstream to both the [Core](https://github.com/open-telemetry/opentelemetry-collector) and [Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib) repos.
 
-  When [PR22839](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22839) on the [OpenTelemetry Collector Contrib repo](https://github.com/open-telemetry/opentelemetry-collector-contrib) has been merged, the documentation below will be updated to reflect the main branch of the Contrib repo.
-</Callout>
+## Step 3: Download or clone the examples repo [#example]
 
-See [https://github.com/4demos/opentelemetry-collector-contrib.git](https://github.com/4demos/opentelemetry-collector-contrib) for latest installation instructions.
+This setup uses the example collector configuration in [New Relic's OpenTelemetry Examples repo](https://github.com/newrelic/newrelic-opentelemetry-examples).
 
-```bash
-git clone https://github.com/4demos/opentelemetry-collector-contrib.git
-cd opentelemetry-collector-contrib
-make otelcontribcol
-
-```
+Download the repo above, and navigate to the [Confluent Cloud example](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/collector/confluentcloud) directory. For more information, you can check the `README` there as well.
 
-The binary will be installed under `./bin`
+## Step 4: Add the pem key files and set variables [#configure-opentelemetry-collectors]
 
-## Step 4: Configure Opentelemetry collectors [#configure-opentelemetry-collectors]
+* Add the TLS authentication keys (key.pem, cert.pem, and ca.pem files) created in the pre-requisites into the `confluentcloud` directory.
 
-Create a new file called `config.yaml` from the example below.
+* "Export the following variables or add them in a `.env` file, then run the `docker compose up` command."
 
-Replace the following keys in the file with your own values:
+```bash
+export NEW_RELIC_API_KEY=<your_api_key>
+export NEW_RELIC_OTLP_ENDPOINT=https://otlp.nr-data.net
+export CLUSTER_ID=<your_cluster_id>
+export CLUSTER_API_KEY=<your_cluster_api_key>
+export CLUSTER_API_SECRET=<your_cluster_api_secret>
+export CLUSTER_BOOTSTRAP_SERVER=<your_cluster_bootstrap_server>
+
+docker compose up
+```
 
-* [Cloud API key](https://docs.confluent.io/cloud/current/monitoring/metrics-api.html#metrics-quick-start)
-	* CONFLUENT_API_ID
-	* CONFLUENT_API_SECRET
 * [Kafka Client API key](https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html#resource-specific-api-keys)
```

Comment:
What context can we add to situate this better? It's unclear to me why the bulleted list is here -- are we letting users know how to get the information needed in the previous code snippet? Are we calling out that they need to update this info?